nBTC/sources/nbtc.move

```diff
@@ -91,6 +93,22 @@ public struct NbtcContract has key, store {
     fees_collected: Balance<NBTC>,
     // mapping a spend_key to related dWallet cap for issue signature
     dwallet_caps: Table<vector<u8>, DWalletCap>,
+    redeem_requests: Table<ID, RedeemRequest>,
+    nbtc_lock: Table<ID, Coin<NBTC>>,
+}
+
+public enum RedeemStatus has copy, drop, store {
+    Lock,
```

Comment:
this sounds wrong, reeedem shouldn't be "locked" . I propose the following statuses:
- signing
- confirming
- confirmed